Library/Homebrew/brew.sh

```diff
@@ -805,6 +805,7 @@ then
   if check-array-membership "${HOMEBREW_COMMAND}" "${NO_INSTALL_FROM_API_COMMANDS[@]}"
   then
     export HOMEBREW_NO_INSTALL_FROM_API=1
+    export HOMEBREW_NO_INSTALL_FROM_API_SET_AUTOMATICALLY=1
```

Comment:
Should we set this only when `HOMEBREW_NO_INSTALL_FROM_API` is not already set by the user?

Reply:
@carlocab Yes, good catch.